wikichange/src/content/content.js

```diff
@@ -255,29 +251,45 @@ const renderItemsBelowGraph = async (creationDate) => {
         let curRevisionId = revisions[0][0];
         const oldRevision = revisions[1];
         let oldRevisionId = oldRevision[0];
-        const oldRevisionDate = oldRevision[1].toLocaleDateString().slice(0, 10);
+        const oldRevisionDate = oldRevision[1].toLocaleDateString("en-US").slice(0, 10);
 
         // Change the revision context box
         updateClosestDate(
             getRevisionPageLink(title, curRevisionId, oldRevisionId).replace(/\s/g, "_"),
             oldRevisionDate
         );
-
         highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId);
-        revisionButton.disabled = false;
     });
+    return [curRevisionId, oldRevisionId];
+};
 
-    revisionButton.addEventListener("click", async () => {
-        try {
-            window.open(getRevisionPageLink(title, curRevisionId, oldRevisionId), "_blank");
-        } catch (err) {
-            console.error(
-                `Error getting revision link between revision ids for inputs title:${title} curRevisionId:${curRevisionId} oldRevisionId:${oldRevisionId}\nError: ${err}`
-            );
-        }
-    });
+/**
+ * Render a simple JS loader by the highlight button
+ */
+const renderLoader = () => {
```

Comment:
very nit but is it possible to move this `renderLoader()` to the same location as the main (line 312) so that we can make sure that there isn't really any change in the loader?